File: curso_c_cpp/c/03_entrada_salida/ejemplos/leer_entero.c
/*
 * Leer un entero con scanf (ojo con el & ).
 * Prueba: printf "25\n" | ./prog
 */
#include <stdio.h>

int main(void) {
    int edad;
    printf("Ingresa tu edad: ");
    scanf("%d", &edad);                 // & = direccion de 'edad'
    printf("El proximo anio tendras %d\n", edad + 1);
    return 0;
}

File: curso_java/03_entrada_salida/ejemplos/LeerEntero.java
/*
 * Leer un entero con Scanner.
 * Prueba: printf "25\n" | java LeerEntero
 */
import java.util.Scanner;

public class LeerEntero {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Tu edad: ");
        int edad = sc.nextInt();

        System.out.println("El proximo anio tendras " + (edad + 1));
        sc.close();
    }
}
